aiida/.gitignore

```diff
@@ -0,0 +1,338 @@
+
```

Comment:
you might want to add the `aiida_config.yaml` file to `.gitignore` since you have the project set up to automatically generate one now?

Reply:
yes, for the moment this makes sense.

It probably won't be the long-term solution but it will do for now.